config/jobs/periodic/kubernetes/perf-tests/perf-tests-kubernetes-periodics.yaml

```diff
@@ -99,22 +91,10 @@ periodics:
     org: kubernetes
     repo: perf-tests
   spec:
-    volumes:
-    - name: powercloud-bot-key
-      secret:
-        defaultMode: 256
-        secretName: bot-ssh-secret
     containers:
     - image: quay.io/powercloud/all-in-one:0.5
       command:
       - /bin/bash
-      volumeMounts:
-      - mountPath: /etc/secret-volume
-        name: powercloud-bot-key
-        readOnly: true
-      envFrom:
-      - secretRef:
-          name: ibm-cloud-credentials
```

Comment:
this is not covered in the `preset-ssh-bot` preset, can you ptal

Reply:
The `preset-ssh-bot` definition has the environment variable `TF_VAR_powervs_api_key` fetched from `ibm-cloud-credentials` secret.

Though the secret `ibm-cloud-credentials` has two env variables namely `TF_VAR_powervs_api_key` and `IBMCLOUD_API_KEY`, only the first one is needed by the jobs where we are replacing the presets.

Also, the tag `envFrom` cannot be used in `presets` definition. Hence we are using `env:` and fetching only the required environment variable.